indico_toolkit/association/line_items.py

```diff
@@ -123,14 +114,14 @@ def assign_row_number(self):
         Adds a row_number:int key/val pair based on bounding box position and page
 
         Updates:
-        self._line_item_predictions (list of dicts): predictions with row_number added
+        self._mapped_positions (list of dicts): predictions with row_number added
```

Comment:
do these get sorted (either before this class or after)?

Reply:
they get sorted when they begin the mapping process in get_bounding_boxes